package.json

```diff
@@ -22,5 +28,45 @@
   "bugs": {
     "url": "https://github.com/esbb48/Kizuna/issues"
   },
-  "homepage": "https://github.com/esbb48/Kizuna#readme"
+  "homepage": "https://github.com/esbb48/Kizuna#readme",
+  "dependencies": {
+    "babel-polyfill": "^6.9.1",
+    "grommet": "^0.6.10",
+    "react": "^15.3.0",
+    "react-dom": "^15.3.0",
+    "react-hot-loader": "^3.0.0-beta.1",
```

Comment:
Strange, `` react-hot-loader `` is a `` devDependency `` in other repo. I'll check what's going on there.